src/Oceananigans.jl

```diff
@@ -54,9 +54,12 @@ export
     δρ,
 
     # Boundary conditions
-    BoundaryConditions,
     BoundaryCondition,
-    Default,
+    CoordinateBoundaryConditions,
+    FieldBoundaryConditions,
+    ModelBoundaryConditions,
+    Periodic,
+    FreeSlip,

```

Comment:
I don't think we need a special `FreeSlip` condition. `FreeSlip` is *either* `Gradient` or `Flux` with `condition=0`. I propose we use a `Flux` condition with `condition=0`.